attributes/default.rb

```diff
@@ -0,0 +1,2 @@
+default['osl-app']['nginx'] = {}
+default['osl-app']['haproxy_ips'] = %w(140.211.9.53 140.211.9.52 140.211.9.50)
```

Comment:
Hardcoding IP's in the attributes file?

Reply:
We don't have an easy way to pull this in unfortunately.